nesspoll/src/components/LoginPage.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { Redirect } from 'react-router-dom';
+import { TextField, RaisedButton } from 'material-ui';
+
+interface LoginState {
+    username: String;
+    password: String;
+    redirect: boolean;
+}
+
+interface LoginProps {
+    user: {
+      username: String;
+      password: String;
+    };
+}
+
+class LoginPage extends React.Component<LoginProps, LoginState> {
```

Comment:
Missing documentation.
What is the purpose off LoginProps vs LoginState?

There is username and password specified in both. What is the difference?

Reply:
I thought that state is state of the components when there is a change and props are data so user which is declared in login.tsx and checked here